scratchpad/components/todo/bindings/PublicDir.ts

```diff
@@ -0,0 +1,14 @@
+import {inject} from '@loopback/core';
+
+const PUBLIC_DIR = 'publicDir';
+
+export class PublicDir {
+  name = PUBLIC_DIR;
+  value() {
+    return path.join(__dirname, '..', '..', '..', 'public'); 
+  }
+}
+
+export const publicDir = () => {
```

Comment:
Let's add comments to show case this code as a way to create specialized decorators as sugar to lower level ones.